enterprise/core-edge/src/test/java/org/neo4j/coreedge/raft/log/segmented/TermsTest.java

```diff
@@ -276,16 +276,35 @@ public void shouldAppendAfterSkip() throws Exception
         assertTermInRange( skipIndex + 1, skipIndex + 20, skipTerm );
     }
 
+    @Test
+    public void shouldMaintainSaneMemoryUseForTerms() throws Exception
```

Comment:
I've gone with reflection here, but now assert on size rather than on not blowing up.